src/components/pages/coach/ReportList/index.jsx

```diff
@@ -75,42 +98,39 @@ class ReportList extends React.Component{
     componentDidMount = () => {
         // Set page title
         document.title = "Beautyreport List";
-        
-        if(this.props.location){
-            if(this.props.location.state){
-                if(this.props.location.state.user.id){
+        if(this.props.match){
+            if(this.props.match.params){
+                if(this.props.match.params.uid){
                     this.setState({
-                        user: this.props.location.state.user,
-                        loading: true
-                    }, () => this.fetchReports(this.props.location.state.user.id));
+                        uid: this.props.match.params.uid,
+                        loading: true,
+                    }, () => this.fetchReports(this.props.match.params.uid));
                 }
             }
         }
     }
 
-    componentWillUnmount = () => {
-        //clearInterval(this.interval);
-    }
-
     fetchReports = (uid) => {
         //this.interval = setInterval(() => this.fetchAllReports(uid), 5000);
         // Fetch data required for creating a report
         this.fetchAllReports(uid);
     }
 
     fetchAllReports = (uid) => {;
-        this.props.client.query({
+        clientNoCache.query({
         query: GET_REPORTS,
         variables: { "id": uid, "token": localStorage.getItem("fprint") }
         }).then(({data}) => {
             if(data.brLatestByUid){
                 console.log("Data");
                 console.log(data);
```

Comment:
# please remove `console` (no-console)

In JavaScript that is designed to be executed in the browser, it's considered a best practice to avoid using methods on `console`. Such messages are considered to be for debugging purposes and therefore not suitable to ship to the client. In general, calls using `console` should be stripped before being pushed to production.

```js
console.log("Made it here.");
console.error("That shouldn't have happened.");
```

Examples of **incorrect** code for this rule:

```js
/* eslint no-console: "error" */

console.log("Log a debug level message.");
console.warn("Log a warn level message.");
console.error("Log an error level message.");
```

Reply:
Noted.
However, the console functions do not exist without reason. It is a really great, fully integrated development tool.